src/workloads/scale/LLTQuery.yml

```diff
@@ -0,0 +1,421 @@
+    cuid: {^RandomInt: {min: 0, max: 100000}}
+    prod: {^RandomInt: {min: 0, max: 10000}}
+    price: {^RandomDouble: {min: 0.0, max: 1000.0}}
+    data: {^Join: {array: ["aaaaaaaaaa", {^FastRandomString: {length: {^RandomInt: {min: 0, max: 10}}}}]}}
+
+  LLTIndexes: &LLTIndexes
+    - keys: {price: 1, ts: 1, cuid: 1}     # PDC
+    - keys: {price: 1, cuid: 1}            # Market
+    - keys: {caid: 1, price: 1, cuid: 1}   # Register
+
+  # Loader Config.
+  LoadThreads: &LoadThreads 4
+  LoadBatchSize: &LoadBatchSize 1000
+
+  # In-memory: Database size works out about 12GB.
+  InitialDocumentCount: &InitialNumDocs 100
+  SecondaryDocumentCount: &SecondaryNumDocs 100
+
+  # WIP make a copy of the workload once these values are tuned correctly and
+  # the workload runs consistently.
+  # Not In-memory: Database size works out about 30GB.
+  # InitialDocumentCount: &InitialNumDocs 49000000
+  # SecondaryDocumentCount: &SecondaryNumDocs 1000000
+  CollectionCount: &CollectionCount 4
+
+  PdcQueryOperation: &PdcQueryOperation
+    OperationName: findOne
+    OperationCommand:
+      Filter: &PdcFilter {price: {'$gte' : {^RandomDouble: { min: 0.0, max: 500.0 }}}}
+      Options: &PdcQueryOperationOptions
+        Hint: price_1_ts_1_cuid_1
+        Comment: PdcQueryOperation
+
+  MarketQueryOperation: &MarketQueryOperation
+    OperationName: findOne
+    OperationCommand:
+      Filter: &MarketFilter {'price': {'$gte' : {^RandomDouble: { min: 0.0, max: 500.0 }}}}
+      Options: &MarketQueryOperationOptions
+        Hint: price_1_cuid_1
+        Comment: MarketQueryOperation
+
+  RegisterQueryOperation: &RegisterQueryOperation
+    OperationName: findOne
+    OperationCommand:
+      Filter: &RegisterFilter {'caid':  {'$gte': {^RandomInt: { min: 0, max: 1000 }}}}
+      Options: &RegisterQueryOperationOptions
+        Hint: caid_1_price_1_cuid_1
+        Comment: RegisterQueryOperation
+
+  # Low Baseline / Benchmark
+  LowGlobalRate: &LowGlobalRate 400 per 1 second
+  LowThreads: &LowThreads 8
+
+  # Low Baseline / Benchmark
+  HighGlobalRate: &HighGlobalRate 800 per 1 second
+  HighThreads: &HighThreads 16
+
+  # Scanner
+  SnapshotScannerShortDuration: &SnapshotScannerShortDuration 2 minutes
+  SnapshotScannerMediumDuration: &SnapshotScannerMediumDuration 10 minutes
+  SnapshotScannerLongDuration: &SnapshotScannerLongDuration 30 minutes
+
+Clients:
+  Default:
+    QueryOptions:
+      socketTimeoutMS: -1
+      maxPoolSize: 200
+  HighPool:
+    QueryOptions:
+      socketTimeoutMS: -1
+      maxPoolSize: 400
+
+# Odd phases do operations, even phases quiesce.
+Actors:
+- Name: InitialLoad
+  Type: Loader
+  ClientName: Loader
+  Threads: *LoadThreads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Threads: *LoadThreads
+        CollectionCount: *CollectionCount
+        Database: *dbname
+        Repeat: 1
+        Document: *Doc
```

Comment:
Do we want all the docs loaded with ^Now or maybe some predefined range of dates?

Reply:
I went for now so that the documents are incrementing. I think it is something we could change but I don't see the need at the moment.